sigpro/primitives/sigpro.SigPro.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Can we move this file to `sigpro/primitives/sigpro/` and rename it as `SigPro.json`? See whether this works or not.